eZ/Publish/Core/MVC/Symfony/Locale/LocaleConverterInterface.php

```diff
@@ -8,7 +8,7 @@
 
 /**
  * Interface for locale converters.
- * eZ Publish uses <ISO639-2/B>-<ISO3166-Alpha2> locale format (mostly, some supported locales being out of this format, e.g. cro-HR).
+ * eZ Platform uses <ISO639-2/B>-<ISO3166-Alpha2> locale format (mostly, some supported locales being out of this format, e.g. cro-HR).
```

Comment:
I really appreciate this, but could you at least split it into separate commits for fast-forward merge? It would be squashed as `EZP-31591: Farsi (Persian) language is missing in internal locale conversion map (#52)` which is not gonna make sense for anyone who's reading this in the future.

Reply:
Fair point, changed in: https://github.com/ezsystems/ezplatform-kernel/pull/52/commits/1958a6470bbef36a692d8b1bf9ebf09989075907